### PR TITLE
guest: emulate UCRT ceilf

### LINKED ISSUE #847: x64: UCRT ceilf import を実装して OLMSmoother を admission/render/cleanup へ進める
## Context

The canonical Apple Silicon Unicorn sweep currently stops OLMSmoother.aex during RENDER with `unsupported Win64 import: api-ms-win-crt-math-l1-1-0.dll!ceilf`.

## Scope

- implement the observed Win64/UCRT `ceilf` contract with the correct XMM ABI and deterministic edge-case behavior
- keep library-qualified routing and structured fail-closed diagnostics
- add focused behavioral tests
- rerun the arm64 Release worker and canonical 10-AEX sweep twice

## Acceptance

- focused and full `aex-guest-worker` tests pass
- Release worker is Mach-O arm64
- OLMSmoother no longer stops at the `ceilf` import
- canonical sweep shows no lost render/cleanup or output changes

CI is outside the merge gate while billing limits prevent Actions from running. Certificate signing and notarization are outside this issue.

### PR DESCRIPTION
**Pull request by onmokoworks**
Closes #847

## What changed

- add library-qualified `api-ms-win-crt-math-l1-1-0.dll!ceilf` routing
- execute the Win64 scalar-float contract through XMM0 using deterministic `f32::ceil` semantics
- keep same-named imports from unrelated DLLs fail-closed
- cover finite values, infinities, NaN, negative zero, routing, and bounded math tracing

## Impact

- OLMSmoother now completes admission, render, and cleanup with `session_clean=true`
- canonical 10-AEX Unicorn coverage improves from `7/7/7` to `8/8/8`

## Validation

- `cargo fmt --check`
- `cargo test -p aex-guest-worker` (239 lib + 9 bin)
- Apple Silicon Release: Mach-O arm64, SHA-256 `545c0b4cf2e7d6c1bd1c5aeeb207756129ae04cd4bbb10d342ffebe3496f28c7`
- sweep 1: `/private/tmp/aexcompat-issue847-1.json`
- sweep 2: `/private/tmp/aexcompat-issue847-2.json`
- gained render: OLMSmoother; no lost render/cleanup or output changes

GitHub Actions remains outside the gate while billing limits prevent CI from running. Certificate signing and notarization are outside this issue.

### CODE REVIEW

**Comment by onmokoworks** [comment]
@codex review

File: guest/crates/aex-guest-worker/src/x64/imports.rs [hunk 1]
```diff
@@ -343,6 +344,8 @@ fn dispatch_win64_import(library: &str, symbol: &str) -> Win64ImportDispatch {
         (_, "getenv") => return Win64ImportDispatch::UnsupportedLegacyImport,
         ("api-ms-win-crt-math-l1-1-0.dll", "cos") => LegacyWin64Import::Cos,
         (_, "cos") => return Win64ImportDispatch::UnsupportedLegacyImport,
+        ("api-ms-win-crt-math-l1-1-0.dll", "ceilf") => LegacyWin64Import::CeilF,
+        (_, "ceilf") => return Win64ImportDispatch::UnsupportedLegacyImport,
```
[the remark this answers is not in the document]
Comment (onmokoworks) [reply guest/crates/aex-guest-worker/src/x64/imports.rs]: Fixed in dcfbc53. `ceilf` now routes through the same bounded emulator for both `api-ms-win-crt-math-l1-1-0.dll` and concrete `ucrtbase.dll`, while unrelated DLLs remain fail-closed. Added explicit routing coverage. All 239 lib + 9 bin tests pass; arm64 Release SHA-256 is fa80bb210f37fdd3f5154bca09e1bb9915a0ed9fb5e4982429845f9380c6ab65; the canonical sweep remains 8/8/8 with no regression or output change.

**Comment by onmokoworks** [comment]
@codex review
